bench/compare-template-compile-time.js

```diff
@@ -34,13 +35,20 @@ function getCompilers() {
 
       const emberPackage = label.substring('glimmer-component'.length);
 
-      buildCompileOptions =
-        require(`${emberPackage}/dist/ember-template-compiler`).compileOptions;
+      // Use the package version if it's legit; otherwise default to using 3.28
+      try {
+        buildCompileOptions =
+          require(`${emberPackage}/dist/ember-template-compiler`).compileOptions;
+      } catch {
+        buildCompileOptions =
+          require(`ember-source-3-28/dist/ember-template-compiler`).compileOptions;
```

Comment:
Mind making a "well named constant" for this? `const FallBackEmberSource`? Some better name?

(the idea being that it explains why this version is "special" / "magical" at the moment)

Reply:
Yes, I like that.

Reply:
Done!